internal/engine/postgresql/pg_catalog.go

```diff
@@ -11646,13 +11646,23 @@ func genPGCatalog() *catalog.Schema {
 			ReturnType: &ast.TypeName{Name: "integer"},
 		},
 		{
-			Name:       "json_build_array",
-			Args:       []*catalog.Argument{},
+			Name: "json_build_array",
+			Args: []*catalog.Argument{
+				{
+					Type:     &ast.TypeName{Name: "any"},
+					Variadic: true,
+				},
+			},
 			ReturnType: &ast.TypeName{Name: "json"},
 		},
 		{
-			Name:       "json_build_object",
-			Args:       []*catalog.Argument{},
+			Name: "json_build_object",
+			Args: []*catalog.Argument{
+				{
+					Type:     &ast.TypeName{Name: "any"},
```

Comment:
Shouldn't you mark Variadic arguments as having defaults because there can b 0 of them otherwise the following would fail.

```
postgres=# select json_build_object();
 json_build_object
-------------------
 {}
(1 row)
```